salt/modules/boto_cloudfront.py

```diff
@@ -247,13 +241,13 @@ def export_distributions(region=None, key=None, keyid=None, profile=None):
                 {"config": config},
                 {"tags": tags},
             ]
-            results["Manage CloudFront distribution {0}".format(name)] = {
+            results["Manage CloudFront distribution {}".format(name)] = {
                 "boto_cloudfront.present": distribution_sls_data,
             }
-    except botocore.exceptions.ClientError as err:
+    except botocore.exceptions.ClientError as exc:
         # Raise an exception, as this is meant to be user-invoked at the CLI
         # as opposed to being called from execution or state modules
-        six.reraise(*sys.exc_info())
+        log.trace("Boto client error: {}", exc)
```

Comment:
No longer reraise the exception?!
```suggestion
        log.trace("Boto client error: %s", exc)
```